projects/igniteui-angular/src/lib/grids/grid-navigation.service.ts

```diff
@@ -335,16 +351,74 @@ export class IgxGridNavigationService {
             && !curRow.childGridsData && (includeSummary || !curRow.summaries);
     }
 
+    public setActiveNode(activeNode: IActiveNode, tag: GridKeydownTargetType) {
```

Comment:
isn't it a better idea to make the tag parameter as optional and if is not passed to make here the check what is the tag parameter for each element, because from I can see from the code above you make a check on a few other places and in this way will be much more clean and easy to maintain.